pagopa/io-app
# fix: [IABT-1471] Update ZenDesk SDKs to support iOS 16

## Short description
This PR updates Zendesks SDKs to support iOS 16 and fixes an issue with the attachment button not working properly after building with Xcode 14.2.
 
## List of changes proposed in this pull request
- Bumps dependency `io-react-native-zendesk`  to `0.3.24`.

## How to test
- Run `cd ios && bundler exec pod install && cd..` and if necessary `cd ios && bundler exec pod cache clean --all && bundler exec pod deintegrate && cd ..`;
- Run the app, login, press the `?` button and open a new ticket. Check if the attachment button works and if you can attach an image.

<details>
<summary>Example</summary>
<img src="https://user-images.githubusercontent.com/12371664/235670927-845b452b-357c-444e-bbae-0ba00228763d.png" height="800">
</details>

**Note: If you're testing on a Simulator you must toggle the virtual keyboard or the attachment menu won't pop-up.**
